modules/store/testing/src/mock_store.ts

```diff
@@ -83,7 +74,7 @@ export class MockStore<T = object> extends Store<T> {
       typeof selector === 'string'
         ? createSelector(
             () => {},
-            (): Result => value
+            (v): Result => value
```

Comment:
Typescript is losing its mind unless it sees an argument in this list.

Looking for help to make it properly detect the right overload for `createSelector` in #3023